L1Trigger/L1THGCal/python/hgcalTriggerPrimitives_cff.py

```diff
@@ -20,10 +20,12 @@
 
 from Configuration.Eras.Modifier_phase2_hgcalV10_cff import phase2_hgcalV10
 from Configuration.Eras.Modifier_phase2_hgcalV11_cff import phase2_hgcalV11
-from L1Trigger.L1THGCal.customTriggerGeometry import custom_geometry_V10, custom_geometry_V11_Imp3
+from Configuration.Eras.Modifier_phase2_hgcalV16_cff import phase2_hgcalV16
+from L1Trigger.L1THGCal.customTriggerGeometry import custom_geometry_V10, custom_geometry_V11_Imp3, custom_geometry_V16_Imp1
 from L1Trigger.L1THGCal.customCalibration import  custom_cluster_calibration_global
 modifyHgcalTriggerPrimitivesWithV10Geometry_ = (phase2_hgcalV10 & ~phase2_hgcalV11).makeProcessModifier(custom_geometry_V10)
 modifyHgcalTriggerPrimitivesWithV11Geometry_ = phase2_hgcalV11.makeProcessModifier(custom_geometry_V11_Imp3)
```

Comment:
In principle this should be changed to 
```python
modifyHgcalTriggerPrimitivesWithV11Geometry_ = (phase2_hgcalV11 & ~phase2_hgcalV16).makeProcessModifier(custom_geometry_V11_Imp3)
```
Not completely sure if that will make a difference (it can be that the V16 process modifier erases the V11 one), but this is to avoid applying the V11 process modifier to V16+ geometries.